### PR TITLE
Added news priority support

### PR DESCRIPTION
**Pull request by ponjoru**
1. CreateBase.sql added properties game and priority
2. Added news priority support
3. Added priority field to swagger.json

### CODE REVIEW

File: DB-scripts/CreateBase.sql [hunk 1]
```diff
@@ -96,7 +96,9 @@ INSERT INTO each_prop (eid, name, type) VALUES (NEXTVAL('each_seq'), 'like', 'li
 commit;
 INSERT INTO each_prop (eid, name, type) VALUES (NEXTVAL('each_seq'), 'location', 'location');
 commit;
-INSERT INTO each_prop (eid, name, type) VALUES (NEXTVAL('each_seq'), 'museum', 'museum');
```
Comment (daniilsavchuk) [inline DB-scripts/CreateBase.sql]: Why did you delete museum property?

File: DB-scripts/CreateBase.sql [hunk 2]
```diff
@@ -96,7 +96,9 @@ INSERT INTO each_prop (eid, name, type) VALUES (NEXTVAL('each_seq'), 'like', 'li
...
+INSERT INTO each_prop (eid, name, type) VALUES (NEXTVAL('each_seq'), 'game', 'game');
+commit;
```
Comment (daniilsavchuk) [inline DB-scripts/CreateBase.sql]: commit here is unnecessary cause it appears at the end of entire file

File: each/Entities/EntityMuseum.py [hunk 3]
```diff
@@ -80,7 +80,8 @@ def update_from_json(cls, data):
         eid = None
```
Comment (daniilsavchuk) [inline each/Entities/EntityMuseum.py]: Why did you make changes here? Do we try to add priority to the news, don't we? Please if you want to refactor other files - create another pull request :)

Comment (daniilsavchuk) [reply each/Entities/EntityMuseum.py]: This comment is for the whole file

File: DB-scripts/CreateBase.sql (hunk 1 above)
Comment (ponjoru) [reply DB-scripts/CreateBase.sql]: Cause it didn't exsist at the screenshot I sent to you. I can add them if you want

Comment (daniilsavchuk) [reply DB-scripts/CreateBase.sql]: You have the correct file
